articles/dev-box/dev-box-faq.yml

```diff
@@ -51,7 +51,15 @@ sections:
           For optimal latency, we recommend creating pools close to the developer’s physical location. For example, if a developer is based in India, we recommend creating a dev box in the Central India region. This is generally thought of as a best practice to improve typing speeds as developers work within their dev boxes.  
 
           Developers primarily spend their time typing and moving their mouse within their Dev Box. We recommend optimizing for typing latency, since these actions will affect development teams the most. 
-          
+      
+      - question: Can I look up which regions I should create my pools?  
+        answer: |
+          Whether you’re an admin creating pools, or a developer selecting from the list available, it is important to optimize for latency and performance. Sometimes, it is difficult to locate the best region for developers to work in.  
```

Comment:
```suggestion
          Whether you’re creating a new pool, or choosing a pool for creating a dev box, it's important to optimize for latency and performance. It can be difficult to determine the optimal region for developers to host their dev boxes.  
```

Reply:
The specific user persona is of less interest here. It's more about setting context.